packages/calcite-components/src/utils/filter.ts

```diff
@@ -9,16 +9,22 @@ export const filter = (data: Array<object>, value: string): Array<any> => {
     The data argument should be an array of objects`);
   }
 
-  const find = (input: object, RE: RegExp) => {
+  const find = (input: object, RE: RegExp, fields?: string[]) => {
     if ((input as any)?.constant || (input as any)?.filterDisabled) {
       return true;
     }
+
     let found = false;
 
-    forIn(input, (val) => {
+    forIn(input, (val: any, key) => {
```

Comment:
Should `input` be typed as `object` and `key` be typed as `string`?

Reply:
Yeah, will update. I set this type because it was showing as "never".

Reply:
hmm I can't change this to object. I need to leave it as `any`.